src/dotless.Core/Parser/Tree/Quoted.cs

```diff
@@ -37,6 +37,11 @@ public Quoted(string value, bool escaped)
             Quote = null;
         }
 
+        protected override Node CloneCore()
+        {
+            return new Quoted(Value, Quote, Escaped);
```

Comment:
I'm assuming you found this issue by processing an actual less file -- do you think you could write a spec test that fails without this fix?

Reply:
Ah, you are absolutely right, added the exact use-case I was running into as a test.